src/utils/polygon.h

```diff
@@ -378,6 +378,10 @@ class Polygons
     {
         paths.push_back(*poly.path);
     }
+    void add(Polygon&& other_poly)
+    {
+        paths.emplace_back(std::move(*other_poly));
+    }
```

Comment:
I'm still not accustomed to using rvalues and move semantics. I should really use it more often.

For example the `add` function below could use a rvalue reference variant.

Errr.. This comment is more of a remark than it is a comment on your code.